src/core/WorkflowContext.ts

```diff
@@ -365,7 +370,7 @@ export class WorkFlowContext {
             configFile = ensureUserPath(this.tsConfig);
         } else {
             url = path.join(this.homeDir, "tsconfig.json");
-            let tsconfig = findFileBackwards(url, appRoot.path);
```

Comment:
It's not related to the context. It's related to the node_modules. It needs to be changed